app/presenters/facet/boolean_presenter.rb

```diff
@@ -0,0 +1,37 @@
+module Facet
+  class BooleanPresenter < FacetPresenter
+    def display(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `display(*)` if you want the method to accept any arguments but don't care about them.